### DISCUSSION TITLE
HoverCard anchor aria

### DISCUSSION THREAD

**Discussion by ivanov-v** [ariakit/ariakit#1215] [Questions] [answered]
Hello. Why the anchor doesn't have a binding attribute **aria-labelledby**?

**Comment by diegohaz** [comment, marked as answer]
Why should it have?

**Reply by ivanov-v** [reply]
For example, a tooltip anchor has this attribute. Why is it not required here?

**Reply by diegohaz** [reply]
The same reason why a dialog/popover/menu button doesn't have this attribute. A hover card is just a dialog that opens on hover. The anchor has its own label (for example, in the default example `@ariakitjs`).

**Reply by ivanov-v** [reply]
Thanks for the explanation. It's just that the customer really wants to copy the text from the tooltips. That's why I use HoverCard.

**Reply by diegohaz** [reply]
I see. In that case you should add the attributes. We'll have a tooltip component built with hovercard at some point.

**Reply by ivanov-v** [reply]
Thanks a lot
